core/templates/dev/head/domain/exploration/StateObjectFactory.js

```diff
@@ -18,40 +18,26 @@
  */
 
 oppia.factory('StateObjectFactory', [
-  'AnswerGroupObjectFactory',
-  function(AnswerGroupObjectFactory) {
+  'AnswerGroupObjectFactory', 'InteractionObjectFactory',
+  function(AnswerGroupObjectFactory, InteractionObjectFactory) {
   var State = function(name, content, interaction, paramChanges) {
     this.name = name;
+    this.classifier_model_id = null;
```

Comment:
Also I thought @Andimeo had handled this in a recent PR. Is this branch up to date with develop, or was this an omission in that PR?